### PR TITLE
64 tasks api

### PR DESCRIPTION
**Pull request by lillie-dae**
### Description

Implements  #64 .

Implement a tasks api endpoint.
/tasks
/tasks/running

### Status
**Ready**

### Types of changes
- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [x] New tests added to cover the changes.
- [x] All tests passed locally.
- [x] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog
- [x] Any new files have copyright headers
- [x] Code coverage above 70%
- [x] TODO items resolved

### CODE REVIEW

File: src/Contracts/Models/WorkflowInstanceTasksUnwindResult.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        public Status Status { get; set; }
+
+        [JsonProperty(PropertyName = "bucket_id")]
+        public string BucketId { get; set; } = string.Empty;
+
+        [JsonProperty(PropertyName = "input_metadata")]
+        public Dictionary<string, string> InputMetaData { get; set; } = new Dictionary<string, string>();
+
+        [JsonProperty(PropertyName = "tasks")]
+        public TaskExecution? Tasks { get; set; }
```
Comment (jackschofield23) [inline src/Contracts/Models/WorkflowInstanceTasksUnwindResult.cs:50]: should this not be "task" as it is a single taskexecution?

File: src/Database/Interfaces/ITasksRepository.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Monai.Deploy.WorkflowManager.Contracts.Models;
+
+namespace Monai.Deploy.WorkflowManager.Database.Interfaces
+{
+    public interface ITasksRepository
+    {
+        Task<long> CountAsync();
```
Comment (jackschofield23) [inline src/Database/Interfaces/ITasksRepository.cs:25]: Can you add comments to these interface method please

File: src/Contracts/Models/WorkflowInstanceTasksUnwindResult.cs (hunk 1 above)
Comment (lillie-dae) [reply src/Contracts/Models/WorkflowInstanceTasksUnwindResult.cs:50]: I tried that unwind function its used for wouldn't unwind it if it wasn't called Tasks

File: src/WorkflowManager/Properties/launchSettings.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "Monai.Deploy.WorkflowManager": {
       "commandName": "Project",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Local",
+        "ASPNETCORE_ENVIRONMENT": "Development",
```
Comment (jackschofield23) [inline src/WorkflowManager/Properties/launchSettings.json:6]: Is this meant to be commited? or just from testing

File: src/WorkflowManager/appsettings.Local.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "WorkloadManagerDatabase": {
-    "ConnectionString": "mongodb://root:rootpassword@localhost:30017",
+    "ConnectionString": "mongodb://root:rootpassword@localhost:27017",
```
Comment (jackschofield23) [inline src/WorkflowManager/appsettings.Local.json:3]: Again

**Review by jackschofield23 — APPROVED** [review]

**Pull request merged**